lib/vagrant-service-manager/binary_handlers/binary_handler.rb

```diff
@@ -146,6 +147,19 @@ def validate_url
 
       private
 
+      def set_proxy_environment
+        return if @machine.config.servicemanager.proxy.nil?
+
+        env_proxy = @machine.config.servicemanager.proxy
+        if @machine.config.servicemanager.proxy_user
+          user = @machine.config.servicemanager.proxy_user
+          password = @machine.config.servicemanager.proxy_password
```

Comment:
> So what if password is empty? Does it work?

No, either it has to be `proxy` or combined `proxy_user` and `proxy_password`. Anyone of `proxy_user`, proxy_password` will fail.